content/en/security_platform/cspm/frameworks_and_benchmarks.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Compliance Findings
+kind: documentation
+further_reading:
+- link: "security_platform/default_rules"
+  tag: "Documentation"
+  text: "Explore default cloud configuration rules"
+- link: "security_platform/cspm/findings"
+  tag: "Documentation"
+  text: "Search and explore compliance findings"
+---
+
+## Overview
+
+Each default rule maps to one or more controls within a compliance framework or industry benchmark. Datadog currently map rules to the following frameworks & benchmarks:
+
+- [CIS AWS Foundations Benchmark v1.3.0*][1]
+- [CIS Docker Benchmark v1.2.0][1]
+- [CIS Kubernetes Benchmark v1.5.1][2]
+- [PCI DSS v3.2.1][3]
+- [AISCA SOC 2][4]
```

Comment:
Let's hide these for now: 

AISCA SOC 2
GDPR
HIPAA
NIST 800-53 revision 5
NIST 800-171